tests/client/db.rs

```diff
@@ -20,11 +20,13 @@ fn create_collection() {
 
     let results = cursor.next_n(5).unwrap();
 
-    let v3_0 = cfg!(feature = "mongodb_3_0");
-    let result_size = if v3_0 { 3 } else { 2 };
+    let db_version = db.version().unwrap();
+    let v3_1 = db_version.major <= 3 && db_version.minor <= 1;
```

Comment:
Have you tested this on the last 3.1 release? I would have thought that 3.1 would have the same results as 3.2 rather than 3.0